app/components/Banner/Banner.scss

```diff
@@ -1,14 +1,16 @@
-.phase-banner {
+@import "../../App.scss";
```

Comment:
No, its just if you want to include global app styles like in this case wrapper. Ultimately everything should be self contained except for layouts, etc.